.grit/patterns/trulens_eval_migration.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Upgrade to TruLens v1.x
+tags: [python, trulens, migration]
+---
+
+# Upgrade to TruLens v1.x
+
+This pattern upgrades code to TruLens v1.x by replacing deprecated imports and class usage.
+
+```grit
+engine marzano(0.1)
+language python
+
+any {
+  `Tru($session)` => `TruSession($session)`,
```

Comment:
don't know anything about grit but I'm guessing it'll substitute imports(?). Can we add usage instructions to the migration guide?

Reply:
by imports do you mean args? And yeah I can add usage instructions. 

I'm also going to open up a PR to the grit standard lib once this is good to hopefully have it runnable by anyone with Grit with the command: 
`grit apply trulens_eval_migration`

See OpenAI's example: https://docs.grit.io/patterns/library/openai#change-openai-import-to-sync

Reply:
standard lib: https://github.com/getgrit/stdlib

Reply:
FYI for the future you don't need to add patterns to the standard library to make them applyable. Just putting it in your repo here would make it runnable with `grit apply github.com/truera/trulens#trulens_eval_migration`.